ledger/participant-integration-api/src/main/scala/platform/store/interning/StringInterningView.scala

```diff
@@ -94,10 +104,17 @@ class StringInterningView(loadPrefixedEntries: LoadStringInterningEntries)
 
   override def update(
```

Comment:
as discussed it seems like the time is right for a change of strategy here. Let's make LoadStringInterningEntries as a parameter to update.

Reply:
Done